Userland/Libraries/LibCore/Version.cpp

```diff
@@ -12,12 +12,16 @@ namespace Core::Version {
 
 ErrorOr<String> read_long_version_string()
 {
+#ifdef AK_OS_SERENITY
     auto uname = TRY(Core::System::uname());
 
     auto const* version = uname.release;
     auto const* git_hash = uname.version;
 
     return String::formatted("Version {} revision {}", version, git_hash);
+#else
+    return String::from_utf8("Version 1.0"sv);
```

Comment:
It would be nice to use the commit version here as well IMO

Reply:
That's a bit harder to get at. At the moment, we encode the git commit into a Kernel header file, rather than a Lagom-friendly one.

Reply:
That's what I had in mind, but still, that would be nice to have access to that in Lagom.
It should be possible to make CMake define a preprocessor macro to do that, isn't it?

Reply:
IMO the value gained from such a feature is pretty negligible in an environment that explicitly guarantees zero backwards compatibility.